src/main/java/org/elasticsearch/discovery/srv/SrvUnicastHostsProvider.java

```diff
@@ -163,15 +162,15 @@ protected Resolver buildResolver(Settings settings) {
     }
 
     protected List<DiscoveryNode> lookupNodes() throws TextParseException {
-        List<DiscoveryNode> discoNodes = Lists.newArrayList();
+	List<DiscoveryNode> discoNodes = new ArrayList<DiscoveryNode>();
```

Comment:
`org.elasticsearch.common.collect.Lists` which was being used, was removed from newer ES versions. I also took a look at [consul-discovery](https://github.com/lithiumtech/elasticsearch-consul-discovery).

I'll remove the tabs :)